packages/components/ovh-shell/README.md

```diff
@@ -0,0 +1 @@
+# ovh-shell
```

Comment:
**nitpick:** Might be great to have some documentation around this package.

Reply:
It will, it's in another PR that will be updated once we have a more stable architecture to avoid changing the docs too many times.